pkg/streamingpromql/testdata/ours/functions.test

```diff
@@ -1183,3 +1183,19 @@ eval range from 0s to 66m step 6m absent_over_time(histo{case=~"(0|a)"}[10m])
   {} 1
 
 clear
+
+# Tests for double_exponential_smoothing to trigger circular buffer boundary wrap around
+
+load 10s
+  http_requests{job="api-server", instance="0", group="production"}	8000-10x1000
+  http_requests{job="api-server", instance="1", group="production"}	0-20x1000
+  http_requests{job="api-server", instance="0", group="canary"}		0+30x1000 300-80x1000
+  http_requests{job="api-server", instance="1", group="canary"}		0-40x1000 0+40x1000
```

Comment:
Could you please add some test cases where the input contains NaN or Inf values?

Reply:
6bdfe23894ddbdf5c7bda875a3288fa8299d35d5